conduit/relays/base.py

```diff
@@ -0,0 +1,184 @@
+import imp
```

Comment:
imp seems to be deprecated?

Reply:
Indeed; importlib requires 4 lines of code to do the same thing apparently, but nonetheless I have replaced imp with it.